app/components/publications/urgency-level-checkbox.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import CONFIG from 'frontend-kaleidos/utils/config';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
+import { tracked } from '@glimmer/tracking';
+
+export default class UrgencyLevelCheckboxComponent extends Component {
+  @service store;
+
+  @tracked urgencyLevels = null;
+
+  constructor() {
+    super(...arguments);
+    this.loadData.perform();
+  }
+
+  @task
+  *loadData() {
+    this.urgencyLevels = yield this.store.query('urgency-level', {});
+  }
+
+  get isUrgent() {
+    const urgencyLevel = this.args.urgencyLevel;
+    return urgencyLevel.uri === CONFIG.URGENCY_LEVELS.spoedprocedure;
+  }
```

Comment:
Dit zou ik als `get` function op het model van `urgency-level` definieren. Dan kan je die gebruiken in de template van de `urgency-level-checkbox.hbs` maar ook in `app/pods/publications/index/template.hbs`